test/accounts_test.go

```diff
@@ -387,79 +541,123 @@ func TestGetPayoutSchedule(t *testing.T) {
 	}
 }
 
-func TestUploadFileAccounts(t *testing.T) {
-	cases := []struct {
-		name        string
-		fileRequest accounts.File
-		checker     func(*common.IdResponse, error)
-	}{
-		{
-			name: "when data is correct then return ID for uploaded file - IMAGE",
-			fileRequest: accounts.File{
-				File:    "./checkout.jpeg",
-				Purpose: common.BankVerification,
-			},
-			checker: func(response *common.IdResponse, err error) {
-				assert.Nil(t, err)
-				assert.NotNil(t, response)
-				assert.Equal(t, http.StatusCreated, response.HttpMetadata.StatusCode)
-				assert.NotNil(t, response.Id)
-				assert.NotNil(t, response.Links)
-			},
+func createEntity() string {
+	r := accounts.OnboardEntityRequest{
+		Reference:      GenerateRandomReference(),
+		ContactDetails: &accounts.ContactDetails{Phone: &accounts.Phone{Number: "2345678910"}},
+		Profile: &accounts.Profile{
+			Urls: []string{"https://www.superheroexample.com"},
+			Mccs: []string{"0742"},
 		},
-		{
-			name: "when data is correct then return ID for uploaded file - PDF",
-			fileRequest: accounts.File{
-				File:    "./checkout.pdf",
-				Purpose: common.BankVerification,
+		Individual: &accounts.Individual{
+			FirstName:         "Bruce",
+			LastName:          "Wayne",
+			TradingName:       "Batman's Super Hero Masks",
+			NationalTaxId:     "TAX123456",
+			RegisteredAddress: Address(),
+			DateOfBirth:       &accounts.DateOfBirth{Day: 5, Month: 6, Year: 1995},
+			Identification:    &accounts.Identification{NationalIdNumber: "AB123456C"},
+		},
+	}
+
+	entity, _ := OAuthApi().Accounts.CreateEntity(r)
```

Comment:
If this returns error, we should do something like:
`assert.Fail(t, "some error message" ...)`

Reply:
We can apply but we don't know the message